classes/privacy/provider.php

```diff
@@ -38,7 +38,7 @@ class provider implements null_provider {
      *
      * @return string
      */
-    public static function get_reason() {
+    public static function get_reason(): string {
```

Comment:
Thanks for this, can you add a space please 😅
```suggestion
    public static function get_reason() : string {
```